uefi/src/proto/network/snp.rs

```diff
@@ -186,13 +186,36 @@ impl SimpleNetwork {
         status.to_result_with_val(|| NonNull::new(tx_buf.cast()))
     }
 
-    /// Place a packet in the transmit queue of a network interface.
+    /// Place a packet in the transmit queue of the network interface.
```

Comment:
Could the changes in this file be moved to a separate PR? I'd like to review them in isolation from the test changes.

Reply:
For the record: opened #1664